vortex-api/src/main/java/mil/army/usace/hec/vortex/temporal/InstantRecordIndexQuery.java

```diff
@@ -0,0 +1,96 @@
+package mil.army.usace.hec.vortex.temporal;
+
+import java.time.ZonedDateTime;
+import java.util.*;
+import java.util.logging.Logger;
+
+final class InstantRecordIndexQuery implements RecordIndexQuery {
```

Comment:
Rename to `InstantaneousRecordIndexQuery`

Reply:
Renamed